lib/plug/static.ex

```diff
@@ -59,6 +59,9 @@ defmodule Plug.Static do
       file system traversals on every request when this plug is mounted
       at `"/"`. Defaults to `nil` (no filtering).
 
+    * `:prefix_only` - like `:only` but does a prefix match instead.
```

Comment:
I tried that but the code is more complicated because I have to split :only and it was harder to document.